### ISSUE TITLE
Questions on the Robustness/Generalizablity of PEFTGuard

### ISSUE THREAD

**Issue by Raytsang123** [Vincent-HKUSTGZ/PEFTGuard#2]
Hi, this is a great work on backdoor scanning. When reading Section 6.5 of the paper, I'm curious why you use adversarial attack techniques (in the parameter space) to evaluate the robutness of PEFTGuard. Why do you need to **restrict the perturbation magnitude** w.r.t. the original backdoored model's weights when conducting adaptive attacks? Moreover, it seems that you first train a backdoored model and then perturb the model to evade the meta detector of PEFTGuard. However, the perturbation strategy seems to only incorporate the evasion loss against PEFTGuard but **ignoring the backdoor training loss** to maintain the attack success rate. I note that in Table 12 of the paper, when the perturbation magnitude is large (e.g., $\epsilon$=1e-3 for FGSM), the adaptive attack achieves a relatively high evasion rate against PEFTGuard. However, you argue that the backdoor ASR decreases significantly. I'm wondering whether it is because you don's add backdoor training loss when conducting adaptive attacks.

Thank you a lot for your time and insightful explanation!
